apps/console/src/extensions/configs/core.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { CoreConfig } from "./models";
+
+export const coreConfig: CoreConfig = {
```

Comment:
Can we call it commonConfig? Just a suggestion.

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/1998/commits/45069b63f42ba836d0306f789a8f212d6f670fa8